addons/website_sale/views/product_views.xml

```diff
@@ -168,12 +168,6 @@
                 <group name="product_template_images" string="Extra Product Media" invisible="not sale_ok">
                     <field name="product_template_image_ids" class="o_website_sale_image_list" context="{'default_name': name}" mode="kanban" add-label="Add a Media" nolabel="1"/>
                 </group>
-                <group string="eCommerce Description">
-                    <field name="description_ecommerce"
-                           placeholder="This note is added to the product page on your eCommerce shop"
-                           nolabel="1"
-                           colspan="2"/>
-                </group>
```

Comment:
did you accidentally remove this?

Reply:
It was intentional, @paju-odoo ask me to do it